src/goto-programs/goto_convert_functions.cpp

```diff
@@ -44,7 +44,9 @@ void goto_convert_functionst::goto_convert(goto_functionst &functions)
       !symbol_pair.second.is_type && !symbol_pair.second.is_macro &&
       symbol_pair.second.type.id() == ID_code &&
       (symbol_pair.second.mode == ID_C || symbol_pair.second.mode == ID_cpp ||
```

Comment:
Is this really recommended by `clang-format`? Can't we put `symbol_pair.second.mode == ID_cpp ||` on a separate line as well?

Reply:
Nope, `clang-format` wants it that way (tried to change it and `clang-format` reverted my edit).